src/applications/disability-benefits/526EZ/config/form.js

```diff
@@ -261,15 +261,9 @@ const formConfig = {
                       pattern:
                         "Full names can only contain letters, numbers, spaces, dashes ('-'), and forward slashes ('/')",
                     },
-                    'ui:required': formData => {
-                      const {
-                        homelessness: homelessOrAtRisk,
-                      } = formData.veteran;
-                      if (homelessOrAtRisk.isHomeless !== true) {
-                        return false;
-                      }
-                      return !!homelessOrAtRisk.pointOfContact.primaryPhone;
-                    },
+                    'ui:required': formData =>
+                      _.get('veteran.homelessness.isHomeless', formData, '') ===
+                      true,
```

Comment:
:roll_eyes: Prettier.

Much neater than the conditional stuff, though. So that's a plus.